apps/web/app/components/RichText/Editor.tsx

```diff
@@ -0,0 +1,58 @@
+import { EditorContent, useEditor } from "@tiptap/react";
+// eslint-disable-next-line import/no-named-as-default
+import StarterKit from "@tiptap/starter-kit";
+import EditorToolbar from "./toolbar/EditorToolbar";
+import { cn } from "~/lib/utils";
+
+interface EditorProps {
+  content?: string;
+  onChange: (value: string) => void;
+  placeholder?: string;
+  id?: string;
+  className?: string;
+}
```

Comment:
**question**: Can we use `types` instead of `interfaces` throughout the application? 
I think it would be more consistent